### ISSUE TITLE
Add sub navigation component to service page

### ISSUE THREAD

**Issue by freben** [spotify/backstage#985]
Implement the top bar, top nav, and information ticket.

The other cards are left out of scope for this ticket.

Depends on #922 
Depends on #983 

![image](https://user-images.githubusercontent.com/3097461/82799519-74bd8e80-9e7a-11ea-9f52-4c33d925fce9.png)

**Comment by nikek** [comment]
I'll split this one up into separate smaller issues.
